rmgpy/solver/liquid.pyx

```diff
@@ -87,15 +101,21 @@ cdef class LiquidReactor(ReactionSystem):
     def convert_initial_keys_to_species_objects(self, species_dict):
         """
         Convert the initial_concentrations dictionary from species names into species objects,
-        using the given dictionary of species.
+        using the given dictionary of species. Also convert the inlet_concentrations dictionary if given.
         """
         initial_concentrations = {}
-        for label, moleFrac in self.initial_concentrations.items():
+        for label, conc in self.initial_concentrations.items():
             if label == 'T':
                 continue
-            initial_concentrations[species_dict[label]] = moleFrac
+            initial_concentrations[species_dict[label]] = conc
         self.initial_concentrations = initial_concentrations
 
+        inlet_concentrations = {}
+        if self.inlet_concentrations:
```

Comment:
Thanks for the comment! I assumed that initial concentrations == inlet concentrations for CSTR previously, but I agree that it will be useful to have the optional inlet concentrations for CSTR too. I will modify the code.